examples/subcurrency/src/main.sw

```diff
@@ -81,32 +59,32 @@ const STORAGE_BALANCES: b256 = 0x00000000000000000000000000000000000000000000000
 
 /// Contract implements the `Token` ABI.
 impl Token for Contract {
-    fn mint(gas_: u64, coins_: u64, asset_id_: b256, args: ParamsMint) {
+    fn mint(receiver: b256, amount: u64) {
         // Note: authentication is not yet implemented, for now just trust params
         // See https://github.com/FuelLabs/sway/issues/195
-        if args.receiver == MINTER {
+        if receiver == MINTER {
             let storage_slot = hash_pair(STORAGE_BALANCES, MINTER, HashMethod::Sha256);
 
             let mut amount = get::<u64>(storage_slot);
```

Comment:
```suggestion
            let mut receiver_amount = get::<u64>(storage_slot);
```
You are shadowing the function param here.  Though using a mutable var and updating it before using it from the original is a bit verbose.  `let amount = get::<u64>(storage_slot) + amount;` would work, or `let receiver_amount = get::<u64>(storage_slot) + amount;` to be a bit clearer.

Reply:
I had to put the suggestion into two blocks, GH won't let a suggestion span a deleted line...